gallery/src/pages/Home.tsx

```diff
@@ -5,81 +5,83 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
+import { NxP, NxH3, NxH4, NxCode } from '@sonatype/react-shared-components';
```

Comment:
I wonder if we should group these in some way so that a single import gets you all of the "convenience components"

Reply:
> I wonder if we should group these in some way so that a single import gets you all of the "convenience components"

What would the value in that be?

Reply:
> > I wonder if we should group these in some way so that a single import gets you all of the "convenience components"
> 
> What would the value in that be?

It would be tedious to be working on something and keep realizing that you haven't added this convenience component or that convenience component, or even remembering what they are all named. Instead just loading something like 'NxConvenienceComponents` and getting all of them.

Reply:
> It would be tedious to be working on something and keep realizing that you haven't added this convenience component or that convenience component, or even remembering what they are all named. Instead just loading something like 'NxConvenienceComponents` and getting all of them.

You'd still have to reference them by name though, e.g. `<NxConvenienceComponents.NxP>`.  Additionally from the outside there isn't that much difference between a convenience component and any other component; they just don't do as much

Reply:
> > It would be tedious to be working on something and keep realizing that you haven't added this convenience component or that convenience component, or even remembering what they are all named. Instead just loading something like 'NxConvenienceComponents` and getting all of them.
> 
> You'd still have to reference them by name though, e.g. `<NxConvenienceComponents.NxP>`. Additionally from the outside there isn't that much difference between a convenience component and any other component; they just don't do as much

Right, so it sounds like the answer to my original question is that there isn't a way that we could import _all_ of the convenience components as simply as `import { nxConvenienceComponents }`

Reply:
> Right, so it sounds like the answer to my original question is that there isn't a way that we could import all of the convenience components as simply as import { nxConvenienceComponents }

Well no matter what, when you go to use one you'd still have to specify _which_ one, which means you'd need to know its name